client/src/global/components/FormattedDate/index.js

```diff
@@ -1,56 +1,66 @@
-import React, { Component } from "react";
+import React, { useMemo } from "react";
 import PropTypes from "prop-types";
-import format from "date-fns/format";
-import parseISO from "date-fns/parseISO";
-import formatDistance from "date-fns/formatDistance";
+import { useSelector } from "react-redux";
+import { format as formatDate, parseISO, formatDistance } from "date-fns";
+import es from "date-fns/locale/es";
+import nl from "date-fns/locale/nl";
 import isDate from "lodash/isDate";
 
-export default class FormattedDate extends Component {
-  static displayName = "FormattedDate";
-
-  static propTypes = {
-    prefix: PropTypes.string,
-    format: PropTypes.string,
-    date: PropTypes.oneOfType([PropTypes.string, PropTypes.instanceOf(Date)])
-  };
+export default function FormattedDate({
+  date,
+  format = "PPP",
+  prefix,
+  suffix = false
+}) {
+  const lng = useSelector(state => state.ui.persistent.locale.language);
+  const locale = useMemo(() => {
+    switch (lng) {
+      case "es":
+        return es;
+      case "nl":
+        return nl;
+      default:
+        return null;
```

Comment:
Why does `en` become `null`? More generally, why is this necessary? I feel like I'm missing something.

Reply:
We should also all have a conversation about how available locales are defined in the client. I'm not sure that English, Dutch, and Spanish were ever determined as the options; I think they were just specified for testing. Ideally someone could define locales for the instance somewhere, but I don't think we'll have that in this first pass. In any case, we should be careful about hard-coding specific options anywhere, as presumably we'll need to remove those before v7 is released.